tools/GraphClust/NSPDK/NSPDK_candidateClusters.xml

```diff
@@ -12,7 +12,12 @@
 
         mkdir -p SVECTOR &&
 
-        cp $data_svector SVECTOR/data.svector &&
+        #for $ins in $data_svector:
+
+							ln -f -s  $ins SVECTOR
```

Comment:
but i don't overwrite, i'm linking the files in SVECTOR directory. no? i mean it is working correct on my local instance